<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vpSuperResDepthViewer</class>
 <widget class="QDialog" name="vpSuperResDepthViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Super Resolution Viewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QProgressBar" name="progressBar"/>
     </item>
     <item>
      <widget class="QPushButton" name="saveStopButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="imageRadio">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Set the render window to display the super resolved image.</string>
       </property>
       <property name="text">
        <string>Super res</string>
       </property>
       <property name="shortcut">
        <string>S</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="bicubicRadio">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Set the render window to display the bicubic image.</string>
       </property>
       <property name="text">
        <string>Bicubic upsample (single image)</string>
       </property>
       <property name="shortcut">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="depthRadio">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Set the render window to display the 3-D depth model.</string>
       </property>
       <property name="text">
        <string>Depth</string>
       </property>
       <property name="shortcut">
        <string>D</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QVTKWidget" name="renderImage" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header location="global">QVTKWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
